core/src/smartcontracts/isi/world.rs

```diff
@@ -407,33 +382,47 @@ pub mod isi {
         Ok(())
     }
 
+    fn prune_unknown_parameters(state_transaction: &mut StateTransaction) {
+        state_transaction
+            .world
+            .parameters
+            .get_mut()
+            .retain(|parameter| {
+                state_transaction
+                    .world
+                    .executor_data_model
+                    .parameters
+                    .contains(&parameter.id)
+            });
```

Comment:
this looks as if it also removes core parameters?

Reply:
Replied in other thread (in sum: there are no core parameters, _yet_)